lib/tasks/update_lineagedb.rake

```diff
@@ -75,8 +75,11 @@ task update_lineage_db: :environment do
   TaxonLineage.where(started_at: current_date).update_all(version_start: current_lineage_version) # rubocop:disable Rails/SkipsModelValidations
   TaxonLineage.where(ended_at: TaxonLineage.column_defaults["ended_at"]).update_all(version_end: current_lineage_version) # rubocop:disable Rails/SkipsModelValidations
 
+  ## Update Elasticsearch index
+  ## TODO: test if this works
```

Comment:
Is this TODO done?

Reply:
Missed this question?

Reply:
Resolved. Following further investigation, it seems the todo is unnecessary -- elasticsearch callbacks will handle it